src/test/java/duke/task/EventTest.java

```diff
@@ -0,0 +1,32 @@
+package duke.task;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import duke.DukeException;
+public class EventTest {
+    @Test
+    public void toFileStringTest() {
+        try {
+            Event event = new Event("test", "16/10/2001 1200", "20/02/2003 1200");
+            assertEquals(event.toFileString(),
+                    "E | 0 | test | 16/10/2001 1200 | 20/2/2003 1200",
+                    "'toFileString()' should return a simplified string to be saved to file");
+        } catch (DukeException e) {
+            // do nothing
```

Comment:
Is there a reason why this catch block is empty? I do notice this in your another test method.